smoke-tests/src/test/java/com/splunk/opentelemetry/OtlpLogsSmokeTest.java

```diff
@@ -37,7 +37,11 @@ public class OtlpLogsSmokeTest extends SmokeTest {
 
   @Override
   protected Map<String, String> getExtraEnv() {
-    return Map.of("OTEL_LOGS_EXPORTER", "otlp");
+    return Map.of(
+        "OTEL_LOGS_EXPORTER",
+        "otlp",
+        "OTEL_INSTRUMENTATION_COMMON_EXPERIMENTAL_CONTROLLER_TELEMETRY_ENABLED",
```

Comment:
Unclear to me why we need this on a logs smoke test. Is it necessary?

Reply:
It is needed because the test application is a spring boot application that generates extra spans when controller telemetry is enabled. This test searches for a span named `WebController.greeting` that is created only when controller telemetry is enabled.